_rules/device-motion-user-interface-7677a9.md

```diff
@@ -30,17 +30,16 @@ The rule applies to any [HTML document][] with an associated [Window object][] t
 
 ## Expectation 1
 
-For each registered [device orientation event][device orientation] or [device motion event][device motion] in the test target, a set of one or more [instruments][instrument] is available in the same [web page][] supporting outcomes indistinguishable of those from the event.
-
-**Note:** The outcomes considered in this expectation are all the outcomes that result from the device motion operated functionality.
+For each registered [device orientation event][device orientation] or [device motion event][device motion] in the test target there exists a set of one or more [instruments][instrument] supporting indistinguishable results of those from the event.
 
 ## Expectation 2
 
-Each [instrument][] from expectation 1 is [visible][].
+For each [instrument][] in the set of [instruments][instrument] from Expectation 1, one of the following is true:
```

Comment:
Minor issue; if there are two instruments to achieve one function, both of them would be applicable. I think you can reformulate this: "for each test target, at least one instrument from expectation 1 ...."

Reply:
I see the point but I think it's a bit more complex to formulate.
Take for example a value that can be changed (up and down) by tilting the phone. There can be only 1 registered event for it, but 2 buttons for covering the feature (one for +, one for -). Then **both** buttons need to pass Expectation 2.

On the other hand, if there are 3 buttons to increase the slider, then, yes, only one of these need to pass Expectation 2.

Which might lead into a rabbit hole if the device motion event has a lot of possible outcomes and components to cover these are doing weird partial grouping that overlap 🙃

Reply:
@WilcoFiers @Jym77 if there are 2 instruments (i1 and i2) that do the same thing we can identify 3 sets that meet expectation 1 (s1 = {i1}, s2 = {i2}, s3 = {i1, i2}). If i1 is "findable" and i2 is not "findable" then s1 would pass expectation and the rule while the other sets would not.

If both i1 and i2 are needed to support the same functionality of the event then we only have s3 and both i1 and i2 need to be "findable".

So, because of the second scenario, we need all instruments to pass expectation 2. But in the first scenario that might cause the rule to fail when s2 and s3 are considered. I think we can address this with an assumption that the instruments in the set (if there is more than one) fulfil different parts of the event's result.

Reply:
I think you're illustrating only part of the problem I see (and this will lead to a solution)…

Expectation 1 can find several sets. Expectation 2 is about "**the** set from Expectation 1", i.e. implicitly assuming that there is only 1.
I think Expectation 2 should be updated to something like "at least one set from Expectation 1". Needs some heavier rephrasing to get the sentence to be correct 😉 This would solve the cases you explain (Expectation 1 finds s1, s2 and s3 and only s1 passes Expectation 2, which is fine) without need for assumption.


(Weird) case where the assumption is not enough:
3 possible results of the event: r₁, r₂ and r₃.
4 instruments: i₁ does r₁, i₂ does r₂ + r₃, i₃ does r₁+r₂ and i₄ does r₃. 
Then there are 2 sets that match the assumption: {i₁, i₂} and {i₃, i₄} but only one of them needs to be findable.
{i₂, i₃} does not match the assumption, but having it findable (while neither i₁ nor i₄ is) is nonetheless good.

With suggested fix, Expectation 1 would find {i₁, i₂}, {i₃, i₄} and {i₂, i₃} and Expectation 2 would want at least one of them to be findable.